src/librustc/middle/stability.rs

```diff
@@ -432,6 +432,18 @@ struct Checker<'a, 'tcx: 'a> {
 }
 
 impl<'a, 'gcx, 'tcx> TyCtxt<'a, 'gcx, 'tcx> {
+    fn skip_stability_check_due_to_privacy(self, def_id: DefId) -> bool {
```

Comment:
I was hoping this could be used to skip the "missing annotation" error in the original crate, within this same file, ensuring that the logic is self-consistent.